### PR TITLE
Changes to the hero section and the nav-bar.

### PR DESCRIPTION
**Pull request by Anurag-Bharati**
Finished working on feature #59

### CODE REVIEW

**Comment by Anurag-Bharati** [comment]
[Preview](https://anurag-bharati.github.io/Drifty/) here.

File: Website/index.html [hunk 1]
```diff
@@ -1,71 +1,100 @@
...
-	<section id="info" class="info">
+
+		</header>
+	</section>
+
+	<section id="contrib" class="info">
 		<h2>More Information About Drifty</h2>
-		<p>It is currently available in CLI (Command Line Interface) mode and has the GUI (Graphical User Interface) version under active development. You may contribute to this project here.</p>
-		<a href="https://github.com/SaptarshiSarkar12/Drifty" class="btn4"><i class="fab fa-github"></i> Contribute to Drifty</a>
+		<p>It is currently available in CLI (Command Line Interface) mode and has the GUI (Graphical User Interface)
```
Comment (SaptarshiSarkar12) [inline Website/index.html:65]:
```suggestion
		<p>It is currently available in CLI (Command Line Interface) mode and the GUI (Graphical User Interface)
```

File: Website/index.html [hunk 2]
```diff
@@ -1,71 +1,100 @@
...
+			version under active development. You may contribute to this project here.</p>
```
Comment (SaptarshiSarkar12) [inline Website/index.html:66]:
```suggestion
			version is under active development. You may contribute to this project here.</p>
```

File: Website/index.html [hunk 3]
```diff
@@ -1,71 +1,100 @@
...
-			<div class="btn2"><a href="#">ABOUT US</a></div>
-			<div class="btn2"><a href="#download">DOWNLOAD</a></div>
+	<div class="texture"></div>
+	<div class="nav-container">
+		<div class="nav-bar">
+			<img class="logo" src="Resources/Driftyicon.webp" alt="Drifty-icon" />
+			<nav>
+				<ul class="nav-links">
+					<li><a href="#hero">Home</a></li>
+					<li><a href="#aboutus">About</a></li>
```
Comment (SaptarshiSarkar12) [inline Website/index.html:26]:
```suggestion
					<li><a href="#aboutus"><button class="button">About</button></a></li>
```

File: Website/index.html [hunk 4]
```diff
@@ -1,71 +1,100 @@
...
+					<li><a href="#download">Download</a></li>
```
Comment (SaptarshiSarkar12) [inline Website/index.html:27]:
```suggestion
					<li><a href="#download"><button class="button">Download</button></a></li>
```

File: Website/index.html [hunk 5]
```diff
@@ -1,71 +1,100 @@
...
+					<li><a href="#contrib">Contribute</a></li>
```
Comment (SaptarshiSarkar12) [inline Website/index.html:28]:
```suggestion
					<li><a href="#contrib"><button class="button">Contribute</button></a></li>
```

File: Website/index.html [hunk 6]
```diff
@@ -74,15 +103,15 @@ <h2>Contact Us</h2>
...
 	</footer>
 
 	<!-- Scripts -->
-	<label><input type="checkbox" class="btn-toggle"></label>
-	<script src="script.js" ></script>
-	</body>
-</html>
+	<script src="script.js"></script>
+</body>
+
```
Comment (SaptarshiSarkar12) [inline Website/index.html:116]:
```suggestion
```

File: Website/index.html [hunk 7]
```diff
@@ -74,15 +103,15 @@ <h2>Contact Us</h2>
...
+</html>
```
Comment (SaptarshiSarkar12) [inline Website/index.html:117]:
```suggestion
</html>

```

File: Website/responsive-nav.css [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+    }
+    .img-layer {
+
+       display: none;
+    
+    }
+    .header-container {
+        transform: none;
+    }
+
```
Comment (SaptarshiSarkar12) [inline Website/responsive-nav.css:138]:
```suggestion
```

File: Website/responsive-nav.css [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+
```
Comment (SaptarshiSarkar12) [inline Website/responsive-nav.css:139]:
```suggestion
```

File: Website/responsive-nav.css [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+
```
Comment (SaptarshiSarkar12) [inline Website/responsive-nav.css:140]:
```suggestion
```

**Review by SaptarshiSarkar12 — CHANGES_REQUESTED** [review]
@Anurag-Bharati Please commit the suggested changes.

**Comment by Anurag-Bharati** [comment]
Hi @SaptarshiSarkar12, Thanks for the review. Let me know if you find any other issues.

File: Website/index.html [hunk 11]
```diff
@@ -1,71 +1,100 @@
...
-			<a href="https://saptarshisarkar12.github.io/Drifty/"><img src="Resources/Driftyicon.webp" id="btn1" alt="Drifty-icon"/></a>
-		</div>
-		<div class="buttons">
-			<div class="btn2"><a href="#contact">CONTACT US</a></div>
-			<div class="btn2"><a href="#">ABOUT US</a></div>
-			<div class="btn2"><a href="#download">DOWNLOAD</a></div>
+	<div class="texture"></div>
+	<div class="nav-container">
+		<div class="nav-bar">
+			<img class="logo" src="Resources/Driftyicon.webp" alt="Drifty-icon" />
```
Comment (SaptarshiSarkar12) [inline Website/index.html:22]:
```suggestion
			<a href="https://saptarshisarkar12.github.io/Drifty/"><img class="logo" src="Resources/Driftyicon.webp" alt="Drifty-icon" /></a>
```

**Review by SaptarshiSarkar12 — CHANGES_REQUESTED** [review]
Please make this change

**Comment by Anurag-Bharati** [comment]
> Please make this change

Done

**Review by SaptarshiSarkar12 — APPROVED** [review]
LGTM 👍. Thanks for contributing 🚀 🚀

**Pull request merged**